### PR TITLE
fix sandbox keygen and update

### PR DESCRIPTION
**Pull request by dushansilva**
Fix sandbox regeneration UI related to https://github.com/wso2/product-apim/issues/5040

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx [hunk 1]
```diff
@@ -164,8 +164,8 @@ class Details extends Component {
                     <div className={classes.contentDown}>
                         <Switch>
                             <Redirect exact from='/applications/:applicationId' to={redirect_url} />
-                            <Route path='/applications/:applicationId/productionkeys' render={() => <TokenManager keyType='PRODUCTION' selectedApp={{ appId: this.state.application.applicationId, label: this.state.application.name }} />} />
-                            <Route path='/applications/:applicationId/sandBoxkeys' render={() => <TokenManager keyType='SANDBOX' selectedApp={{ appId: this.state.application.applicationId, label: this.state.application.name }} />} />
+                            <Route path='/applications/:applicationId/productionkeys' component={() => <TokenManager keyType='PRODUCTION' selectedApp={{ appId: this.state.application.applicationId, label: this.state.application.name }} />} />
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx]: Shall we destruct the state variables in the `render` method body and use the destruct variable names here.

i:e 

```javascript
const { application } = this.state;
if(application)
const { applicationId, applicationId } = application;
```

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx [hunk 2]
```diff
@@ -139,7 +139,7 @@ class Details extends Component {
     render() {
         const redirect_url = '/applications/' + this.props.match.params.application_uuid + '/productionkeys';
 
-        const { classes, theme } = this.props;
+        const { classes, theme, match } = this.props;
```
Comment (ruwiniwj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx]: This deconstruction can be moved to the top of the render method and 'const match' be used in 'redirect_url ' and ' <InfoBar applicationId={this.props.match.params.application_uuid}'

Comment (dushansilva) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx]: fixed

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx (hunk 1 above)
Comment (dushansilva) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx]: destruct done, do not need the if condition as it is checked before the render is called

Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx]: Ack :+1:

**Pull request merged**